thunderfish/dataloader.py

```diff
@@ -43,49 +44,49 @@ def load_data(filepath, channel=0, verbose=0) :
         unit (string): the unit of the data
     """
     # check types:
-    if not isinstance(filepath, basestring) :
```

Comment:
Actually this type checking stuff should be completely removed. This is matlab style but not python style.